igor-web/src/test/groovy/com/netflix/spinnaker/igor/build/artifact/decorator/ConfigurableFileDecoratorSpec.groovy

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2017 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.build.artifact.decorator
+
+import com.netflix.spinnaker.igor.build.model.GenericArtifact
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class ConfigurableFileDecoratorSpec extends Specification {
+    @Unroll
+    def "decorate artifacts using regex groups"() {
+        when:
+        GenericArtifact genericArtifact = new GenericArtifact(reference, reference, reference)
+        ConfigurableFileDecorator configurableFileDecorator = new ConfigurableFileDecorator(artifactType, decoratorRegex, decoratorRegex)
+        configurableFileDecorator.decorate(genericArtifact)
+
+        then:
+        genericArtifact.name    == name
+        genericArtifact.version == version
+        genericArtifact.type    == type
+
+        where:
+        artifactType    || decoratorRegex                                 || reference                || name            || version || type
```

Comment:
Here should probably be:
`artifactType    | decoratorRegex                                 | reference                || name            | version | type`